### PR TITLE
add ipydrawio 1.0.1

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
  - [x] all packages covered under main `Apache-2.0` 
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Notes:
- there are some clobber warnings between the two packages, will patch here and fix upstream

### CODE REVIEW

**Comment by bollwyvl** [comment]
Hm, don't know what to do about the long windows paths...

**Comment by bollwyvl** [comment]
Going to try working the path length stuff upstream... basically impossible to patch from the sdist alone...

**Comment by bollwyvl** [comment]
I shaved about 30 characters off... some of the paths. We'll see how it goes!

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review!

File: recipes/ipydrawio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+about:
+  home: https://github.com/deathbeds/{{ name }}
+  summary: Draw.io Diagrams as Jupyter Widgets
+  license: Apache-2.0
+  license_file: {{ name }}/LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
```
Comment (isuruf) [inline recipes/ipydrawio/meta.yaml:124]:
```suggestion
    - bollwyvl
    feedstock-name: ipydrawio
```

File: recipes/ipydrawio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
+{% set name = "ipydrawio" %}
+{% set version = "1.0.1" %}
+
+{% set build_number = 0 %}
+
+package:
+  name: {{ name }}-build
```
Comment (isuruf) [inline recipes/ipydrawio/meta.yaml:7]: Can you do a `s/{{ name }}/ipydrawio/g` for this file? It's hurting readability.

File: recipes/ipydrawio/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/ipydrawio/meta.yaml:123]: 876af84

File: recipes/ipydrawio/meta.yaml (hunk 2 above)
Comment (bollwyvl) [reply recipes/ipydrawio/meta.yaml]: a9534c2

**Comment by bollwyvl** [comment]
Thanks for the speedy review! Have updated from comments...

**Comment by isuruf** [comment]
What's the reason for having one recipe instead of 2 for two different packages?

**Comment by bollwyvl** [comment]
> one recipe instead of 2 for two different packages?

It's to avoid the heavy dependencies (node) on the pdf export side. They are both cut from the same tag... in an ideal world they would be the same (python) package, but I tire of `extras`... and not everybody's on conda :crying_cat_face:

**Comment by bollwyvl** [comment]
Thanks!

**Pull request merged**